lib/components/translucent_text_field/translucent_text_field.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_platform_widgets/flutter_platform_widgets.dart';
+
+class TranslucentTextField extends StatelessWidget {
+  const TranslucentTextField({
+    Key? key,
+    this.placeholder,
+    this.after,
+    this.keyboardType = TextInputType.text,
+    this.obscureText = false,
+    this.controller,
+  }) : super(key: key);
+
+  final String? placeholder;
+  final Widget? after;
```

Comment:
we could rename this widget to be clearer, how about `trailing`? Like `ListTile(trailing: )` 🙏